.github/workflows/codebuild-ci.yml

```diff
@@ -0,0 +1,46 @@
+name: PR Checks
+on:
+    pull_request:
+
+concurrency:
+    group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+    cancel-in-progress: true
+
+permissions:
+    id-token: write # This is required for requesting the JWT
+
+jobs:
```

Comment:
It only have unit-tests and docstyle projects since only for PRs? What is the plan for integs?

Reply:
We are working on a split for our integs so will add those in when it is finalized and tested. Until then will rely on old integs currently being used. We will phase those out in coming PRs.